src/js/test/public/SimpleTransform.test.ts

```diff
@@ -0,0 +1,60 @@
+import { createRecordBatch, SimpleTransform } from "../../modules/public";
```

Comment:
Same rational here

Reply:
Does the previous comment solve this comment?